src/Styleguide/Pages/Artist/Routes/RelatedArtists/RelatedArtistsQuery.ts

```diff
@@ -0,0 +1,14 @@
+import { graphql } from "react-relay"
+
+export const RelatedArtistsQuery = graphql`
+  query RelatedArtistsQueryRendererQuery(
+    $artistID: String!
+    $first: Int!
+    $kind: RelatedArtistsKind!
+  ) {
+    artist(id: $artistID) {
+      ...RelatedArtistsRefetchContainer_artist
+        @arguments(kind: $kind, first: $first)
+    }
+  }
+`
```

Comment:
Same

Reply:
This also applies to comment above -- Under related artists tab there are a few different sub-sections being rendered out, but the [initial fetch](https://github.com/damassi/reaction/blob/776e8a0a59e6f0cc06a41185d78e5a3202638d82/src/Styleguide/Pages/Artist/routes.tsx#L102) takes place on the router. If we can collapse everything to there then we can remove this shared query and the QueryRenderer.

Reply:
👍